d7/Dockerfile

```diff
@@ -1,5 +1,16 @@
 FROM mhart/alpine-node:7
 
+ARG BUILD_DATE
+ARG VCS_REF
+
+LABEL org.label-schema.build-date=$BUILD_DATE \
+  org.label-schema.vcs-ref=$VCS_REF \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.name="docker-gulp" \
```

Comment:
Why docker-gulp?
Regarding description on this page http://label-schema.org/rc1/#label-semantics
`Text description of the image. May contain up to 300 characters.`